src/index.html

```diff
@@ -9,6 +9,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="logo">
+          <div class="logo__img"> <a href="#home"><img src="images/logo.png" alt="moyo-logo"></a></div>
+        </div>
+        <nav class="menu">
+          <a href="#" class="menu__item is-active">Apple</a>
+          <a href="#" class="menu__item">Samsung</a>
+          <a href="#" class="menu__item">Smartphones</a>
+          <a href="#" class="menu__item"
```

Comment:
Please re-read point 13 in checklist
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/ed30656b-0ed7-46dc-b032-afd5f6ae4a03)

Reply:
not fixed